packages/connector-puppeteer/src/lib/chromium-finder.ts

```diff
@@ -337,7 +337,8 @@ const resolveChromiumPath = () => {
 
 /**
  * Searchs for a valid Chromium browser from the ones supported. The current priority list is:
- * `Chrome Canary`, `Chrome`, `Chromium`, `Edge Canary`, `Edge Dev` (`Edge` only on `win32 platforms).
+ * `Puppeteer`, `Chrome Canary`, `Chrome`, `Chromium`, `Edge Canary`, `Edge Dev` (`Edge` only
+ * on `win32 platforms).
```

Comment:
Now that it is also in mac we should fix this...

Reply:
Good point - didn't even notice that part of the comment. I assume there's work to do here to find the right path so we'll probably want to log a bug.